manifests/kubo.yml

```diff
@@ -117,3 +117,20 @@ variables:
   type: password
 - name: kubelet-password
   type: password
+- name: kubo_ca
```

Comment:
I got confused when saw this extra `kubo-ca` certificate. But if this works with CredHub, I am fine.